src/snyk/common/languageServer/languageServer.ts

```diff
@@ -126,6 +125,7 @@ export class LanguageServer implements ILanguageServer {
       ...settings,
       integrationName: CLI_INTEGRATION_NAME,
       integrationVersion: await Configuration.getVersion(),
+      deviceId: this.user.anonymousId,
```

Comment:
why are these 4 fields not part of the settings object?

Reply:
added an explanation comment 👍